Userland/Libraries/LibWeb/SVG/SVGGraphicsElement.cpp

```diff
@@ -116,42 +116,32 @@ Gfx::AffineTransform SVGGraphicsElement::get_transform() const
     return transform;
 }
 
+constexpr Array s_attribute_style_properties {
+    // FIXME: The `fill` attribute and CSS `fill` property are not the same! But our support is limited enough that they are equivalent for now.
+    CSS::PropertyID::Fill,
+    // FIXME: The `stroke` attribute and CSS `stroke` property are not the same! But our support is limited enough that they are equivalent for now.
+    CSS::PropertyID::Stroke,
+    CSS::PropertyID::StrokeWidth,
+    CSS::PropertyID::FillRule,
+    CSS::PropertyID::FillOpacity,
+    CSS::PropertyID::StrokeOpacity,
+    CSS::PropertyID::Opacity,
+    CSS::PropertyID::TextAnchor,
+    CSS::PropertyID::FontSize,
+    CSS::PropertyID::Mask
+};
```

Comment:
Maybe we can put these in a struct that has both the `CSS::PropertyID` and the corresponding `FlyString` attribute name? Then we don't have to call `CSS::string_from_property_id` repeatedly :)

Reply:
Done :+1: